### PR TITLE
app-crypt/heimdal: libressl support

### PR DESCRIPTION
**Pull request by soredake**

### CODE REVIEW

File: app-crypt/heimdal/heimdal-7.1.0.ebuild [hunk 1]
```diff
@@ -15,9 +15,13 @@ SRC_URI="https://github.com/${PN}/${PN}/releases/download/${P}/${P}.tar.gz"
 LICENSE="BSD"
 SLOT="0"
 KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~mips ~ppc ~ppc64 ~s390 ~sh ~sparc ~x86 ~amd64-fbsd"
-IUSE="afs +berkdb caps hdb-ldap ipv6 otp +pkinit selinux ssl static-libs test X"
+IUSE="afs +berkdb caps hdb-ldap ipv6 otp +pkinit selinux libressl ssl static-libs test X"
```
Comment (mingzym) [inline app-crypt/heimdal/heimdal-7.1.0.ebuild]: is IUSE shorted in most case?

Comment (mgorny) [inline app-crypt/heimdal/heimdal-7.1.0.ebuild]: @mingzym, did you mean 'sorted'? And yes, @soredake, this ought to be sorted.

**Comment by mgorny** [comment]
I presume you've tested it against libressl. Did you test a few flag combinations? I think it'd be reasonable to check with all of the USE flags enabled, or at least all of the flags that might be relevant to ssl/crypto properties.

**Comment by mingzym** [comment]
heimdal is able to build with libressl, with this patch, I can commfirm this patch will work to put heimdal and libressl in my gentoo. so far so good.

**Comment by mgorny** [comment]
@gentoo/libressl

File: app-crypt/heimdal/heimdal-7.1.0.ebuild (hunk 1 above)
Comment (soredake) [inline app-crypt/heimdal/heimdal-7.1.0.ebuild]: sorted.

File: app-crypt/heimdal/heimdal-7.1.0.ebuild [hunk 2]
```diff
@@ -15,9 +15,13 @@ SRC_URI="https://github.com/${PN}/${PN}/releases/download/${P}/${P}.tar.gz"
 LICENSE="BSD"
 SLOT="0"
 KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~mips ~ppc ~ppc64 ~s390 ~sh ~sparc ~x86 ~amd64-fbsd"
-IUSE="afs +berkdb caps hdb-ldap ipv6 otp +pkinit selinux ssl static-libs test X"
+IUSE="+berkdb +pkinit X afs caps hdb-ldap ipv6 libressl otp selinux ssl static-libs test"
```
Comment (mingzym) [inline app-crypt/heimdal/heimdal-7.1.0.ebuild]: I think that sort means "afs +berkdb caps hdb-ldap ipv6 libressl  otp +pkinit selinux ssl static-libs test X" is the way , it is way too confusing, aha.

Comment (soredake) [inline app-crypt/heimdal/heimdal-7.1.0.ebuild]: Sorry, i've just used sort command from vim, i'll fix it right now.

**Comment by soredake** [comment]
bump

**Comment by soredake** [comment]
@monsieurp applied.

**Pull request closed without merging**